golang/cosmos/daemon/cmd/root.go

```diff
@@ -158,8 +158,6 @@ func initRootCmd(sender Sender, rootCmd *cobra.Command, encodingConfig params.En
 		txCommand(),
 		keys.Commands(gaia.DefaultNodeHome),
 	)
-	// add rosetta
-	rootCmd.AddCommand(server.RosettaCommand(encodingConfig.InterfaceRegistry, encodingConfig.Marshaler))
```

Comment:
Why remove Rosetta?  Isn't this unrelated to the PFM integration?

Reply:
Yes unrelated. Not entirely sure why. I think to rapidly get it out of the way while trying to isolate a failure in debugging. Not meant to be in the final PR.

Reply:
restoring the file to original